packages/camera/camera_android_camerax/AUTHORS

```diff
@@ -0,0 +1,4 @@
+# Below is a list of people and organizations that have contributed
+# to the Flutter project. Names should be added to the list like so:
+#
+#   Name/Organization <email address>
```

Comment:
This should have `Google Inc.` as a line.

Reply:
@stuartmorgan Thanks for taking a look! I'm working on the first related PR and I'll make these changes there.